activities/views/activity_detail.py

```diff
@@ -59,26 +123,21 @@ def put(self, request: HttpRequest, *args: Any, **kwargs: Any) -> response.Respo
             if res:
                 return res
 
-        attachment_ids_to_remove = request.data.get("remove_attachments", [])
+        return None
 
-        if attachment_ids_to_remove:
-            image_deleter(attachment_ids_to_remove)
+    def __kick_attendee(self, request: HttpRequest) -> None:
+        """Handle kick attendee from activity.
 
-        attachment_to_add = request.data.get("new_images", [])
-        if attachment_to_add:
-            if any("base64" in attachment for attachment in attachment_to_add):
-                image_loader_64(attachment_to_add, activity)
-            else:
-                image_loader(attachment_to_add, activity)
+        :param request: HttpRequest object
+        """
+        activity = self.get_object()
 
-        activity.refresh_from_db()
+        attendee_ids_to_remove = request.data.get("attendee_to_remove", [])
+        attendee_to_remove = activity.attend_set.filter(user__id__in=attendee_ids_to_remove, is_host=False)
 
-        return response.Response(
-            {
-                "message": f"You have successfully edited the activity {res_dict.get('name')}",
-                "id": res_dict.get("id")
-            }
-        )
+        print(attendee_ids_to_remove)
+        attendee_to_remove.delete()
+        return
```

Comment:
It is pointless to have a return (which didn't return anything at the end of the method.